setup/setup.md

```diff
@@ -64,32 +65,34 @@ var azureAppId = "<<add AppId here>>";
 // (2) Planner task's tenant-specific base URL. Get this from Planner with an open task.
 var plannerTaskUrl = "<<Planner URL goes here>>";
 
-``` 
+```
 
-3. Using the two numbered items in your Notepad file, update the initialization values for these two variables.
+1. Using the two numbered items in your text file, update the initialization values for these two variables.
 
-# You are now ready to build the Proseware Tasks sample add-in!
+## You are now ready to build the Proseware Tasks sample add-in
 
-1. In Visual Studio, right click on "taskerWeb" project name in Solution Explorer and select "**Set as StartUp Project**":
+1. In Visual Studio, right click on the `taskerWeb` project name in **Solution Explorer** and select **Set as StartUp Project**.
 
 ![Set Web Project as Startup](images/setasstartupproject.png)
 
-2. Right click on taskerWeb project again and select "**Properties**". Then select Web | Start action | Specific page and insert: Home.html
+1. Right click on `taskerWeb` project again and select **Properties**. Then select **Web** | **Start action** | **Specific page** and insert the following: `Home.html`
+
+    ![Set Startup action to Specific page](images/specificpage.png)
 
-![Set Startup action to Specific page](images/specificpage.png)
+1. Press **Ctrl-F5** to start without debugging. You should see the `Home.html` **Sign in** button.
```

Comment:
When I first did this, I got a prompt saying 

> This project is configured to use SSL. To avoid SSL warnings in the browser you can choose to trust the self-signed certificate that IIS Express has generated.

I had to accept and install the cert that IIS express generated. Should we add something about that here?

Reply:
yes, we can add a note so participants aren't confused here.

Reply:
I already merged, just submit the changes in a new PR